incubator/cassandra/templates/cassandra-statefulset.yaml

```diff
@@ -105,6 +105,13 @@ spec:
         volumeMounts:
         - name: data
           mountPath: /var/lib/cassandra
+        {{- if not .Values.persistence.enabled }}
```

Comment:
you may want a second toggle so that a user who doesn't care at all about data-loss doesn't have a penalty of `terminationGracePeriodSeconds` when terminating pods. I guess they could set `terminationGracePeriodSeconds` to `1` or something low, but would be nice to be able to just skip it altogether.

Reply:
The default is 30 seconds anyway (https://v1-8.docs.kubernetes.io/docs/api-reference/v1.8/#podspec-v1-core) so this is fine. It can be set to 0 if someone really wants to skip it.